### PR TITLE
license header removed

### PR DESCRIPTION
**Pull request by gravitydestroyer**
CodeTemplates.xml did not have any license header.
License header from the rest two files has been removed. :)

### CODE REVIEW

**Comment by emilianbold** [comment]
You don't have to remove, but replace them with the Apache license.

For XML files, copy-paste a license header from another XML file, for example: https://github.com/emilianbold/incubator-netbeans/blob/master/xml.jaxb/build.xml

For .properties file, copy-paste a license header from another .properties file, for example:
https://github.com/emilianbold/incubator-netbeans/blob/master/xml.jaxb/src/org/netbeans/modules/xml/jaxb/Bundle.properties

**Comment by emilianbold** [comment]
Also, your commit message must be "[NETBEANS-54] Module Review xml.jaxb"

File: xml.jaxb/src/org/netbeans/modules/xml/jaxb/layer.xml [hunk 1]
```diff
@@ -1,49 +1,21 @@
...
-Version 2 license, then the option applies only if the new code is
-made subject to such option by the copyright holder.
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
```
Comment (emilianbold) [inline xml.jaxb/src/org/netbeans/modules/xml/jaxb/layer.xml]: Compared with the header here https://github.com/emilianbold/incubator-netbeans/blob/master/xml.jaxb/build.xml you are missing some new lines. Just copy-paste the license header, don't reformat it.

**Comment by emilianbold** [comment]
BTW, note https://github.com/emilianbold/incubator-netbeans/issues/219 which was auto-generated afterwards and mentions some test files too.

File: xml.jaxb/src/org/netbeans/modules/xml/jaxb/layer.xml [hunk 2]
```diff
@@ -1,49 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE filesystem PUBLIC "-//NetBeans//DTD Filesystem 1.2//EN" "http://www.netbeans.org/dtds/filesystem-1_2.dtd">
```
Comment (emilianbold) [inline xml.jaxb/src/org/netbeans/modules/xml/jaxb/layer.xml:2]: This seems to move the DOCTYPE.

File: xml.jaxb/src/org/netbeans/modules/xml/jaxb/ui/Bundle.properties [hunk 3]
```diff
@@ -1,40 +1,20 @@
...
-# License.  When distributing the software, include this License Header
-# Notice in each file and include the License file at
-# nbbuild/licenses/CDDL-GPL-2-CP.  Oracle designates this
-# particular file as subject to the "Classpath" exception as provided
-# by Oracle in the GPL Version 2 section of the License file that
-# accompanied this code. If applicable, add the following below the
-# License Header, with the fields enclosed by brackets [] replaced by
-# your own identifying information:
-# "Portions Copyrighted [year] [name of copyright owner]"
+
```
Comment (emilianbold) [inline xml.jaxb/src/org/netbeans/modules/xml/jaxb/ui/Bundle.properties]: This seems to have a extra empty line.

**Review by emilianbold — CHANGES_REQUESTED** [review]

**Comment by emilianbold** [comment]
@gravitydestroyer let me know if you still are working on this.

**Comment by emilianbold** [comment]
@gravitydestroyer if I don't get a reply this week I'm going to assume you abandoned the issue.

**Comment by gravitydestroyer** [comment]
@emilianbold checkout my latest commit.Sorry for the inconvenience.

**Review by emilianbold — CHANGES_REQUESTED** [review]
I see layer.xml was already modified upstream to have the license header so you don't have to do anything (please revert your change).

Only xml.jaxb/src/org/netbeans/modules/xml/jaxb/ui/Bundle.properties needs to be updated apparently.

**Comment by emilianbold** [comment]
@gravitydestroyer you are almost done! Just undo you change in xml.jaxb/src/org/netbeans/modules/xml/jaxb/layer.xml.

Do you need any help?

**Comment by emilianbold** [comment]
Hacktoberfest is over. Closing this PR. Feel free to reopen one for the issue, I will merge it if it's ok.

**Pull request closed without merging**